state-chain/pallets/cf-environment/src/lib.rs

```diff
@@ -306,20 +347,68 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Manually witnesses the current Bitcoin block number to complete the pending vault
+		/// rotation.
+		///
+		/// ## Events
+		///
+		/// - [BitcoinBlockNumberSetForVault](Event::BitcoinBlockNumberSetForVault)
+		///
+		/// ## Errors
+		///
+		/// - [BadOrigin](frame_support::error::BadOrigin)
+		#[allow(clippy::too_many_arguments)]
+		#[pallet::call_index(5)]
+		// This weight is not strictly correct but since it's a governance call, weight is
+		// irrelevant.
+		#[pallet::weight(Weight::zero())]
+		pub fn witness_initialize_arbitrum_vault(
+			origin: OriginFor<T>,
+			block_number: u64,
+			key_manager_address: cf_chains::evm::Address,
+			vault_address: cf_chains::evm::Address,
+			address_checker_address: cf_chains::evm::Address,
+			chain_id: cf_chains::evm::api::EvmChainId,
+			arb_usdc_address: cf_chains::evm::Address,
+		) -> DispatchResultWithPostInfo {
+			T::EnsureGovernance::ensure_origin(origin)?;
+
+			use cf_traits::VaultKeyWitnessedHandler;
+
+			// Witness the agg_key rotation manually in the vaults pallet for bitcoin
+			let dispatch_result =
+				T::ArbitrumVaultKeyWitnessedHandler::on_first_key_activated(block_number)?;
+
+			ArbitrumChainId::<T>::put(chain_id);
+			ArbitrumAddressCheckerAddress::<T>::put(address_checker_address);
+			ArbitrumVaultAddress::<T>::put(vault_address);
+			ArbitrumKeyManagerAddress::<T>::put(key_manager_address);
+			ArbitrumSupportedAssets::<T>::insert(ArbAsset::ArbUsdc, arb_usdc_address);
```

Comment:
If we we're only supporting arbusdc we don't need the arbeth witneser - in fact, why even have ArbEth as an asset in the enum?

Reply:
As mentioned above, we will support ArbEth